package.json

```
@@ -2,7 +2,7 @@
   "name": "azure-rest-api-specs",
   "devDependencies": {
     "@azure-tools/typespec-apiview": "0.4.4",
-    "@azure-tools/typespec-autorest": "0.36.0",
+    "@azure-tools/typespec-autorest": "https://artprodcus3.artifacts.visualstudio.com/A0fb41ef4-5012-48a9-bf39-4ee3de03ee35/29ec6040-b234-4e31-b139-33dc4287b756/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL2F6dXJlLXNkay9wcm9qZWN0SWQvMjllYzYwNDAtYjIzNC00ZTMxLWIxMzktMzNkYzQyODdiNzU2L2J1aWxkSWQvMzI1OTEzNy9hcnRpZmFjdE5hbWUvcGFja2FnZXM1/content?format=file&subPath=%2Fazure-tools-typespec-autorest-0.37.0-pr-3895.20231113.28.tgz",
```

```suggestion
    "@azure-tools/typespec-autorest": "https://artprodcus3.artifacts.visualstudio.com/A0fb41ef4-5012-48a9-bf39-4ee3de03ee35/29ec6040-b234-4e31-b139-33dc4287b756/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL2F6dXJlLXNkay9wcm9qZWN0SWQvMjllYzYwNDAtYjIzNC00ZTMxLWIxMzktMzNkYzQyODdiNzU2L2J1aWxkSWQvMzI1OTE0OS9hcnRpZmFjdE5hbWUvcGFja2FnZXM1/content?format=file&subPath=%2Fazure-tools-typespec-autorest-0.37.0-pr-3905.20231113.29.tgz",
```

testing this hotfix https://github.com/Azure/typespec-azure/pull/3905